### PR TITLE
Add Affiliate#update support

### PR DESCRIPTION
**Pull request by wowawiwa**

### CODE REVIEW

**Comment by antlypls** [comment]
@psychocandy do you know who and why added circleci to the project?

**Comment by miry** [comment]
@antlypls feel free to remove it. I think circle ci has a bit faster response than travis ci.

File: lib/hasoffersv3/affiliate.rb [hunk 1]
```diff
@@ -9,6 +9,12 @@ def find_by_id(params = {})
       get_request 'findById', params
     end
 
+    def update(params = {})
+      requires! params, [:id]
+      requires! params, [:data]
```
Comment (miry) [inline lib/hasoffersv3/affiliate.rb]: @wowawiwa is it possible to specify in one line requires params?

**Comment by psychocandy** [comment]
@antlypls I guess it was @miry.

👍

**Comment by miry** [comment]
circle ci is green. feel free to merge @antlypls

File: lib/hasoffersv3/affiliate.rb (hunk 1 above)
Comment (wowawiwa) [inline lib/hasoffersv3/affiliate.rb]: @miry fixed

**Pull request merged**